book/special.qmd

```diff
@@ -851,188 +851,188 @@ The experiment shows that using the K-means algorithm with four centers has the
+% There is a clear separation between coordinates using the 'correct' spatial resampling method whereas the train and test splits overlap significantly (and are therefore highly correlated) using the non-spatial method.
+%
+% ```{r special-047}
+% #| fig-cap: Scatterplots show separation of train (blue) and test (orange) data for the first three (left to right) folds of the first repetition of the cross-validation. The top row is spatial resampling where train and test data are clearly separated. The bottom row is non-spatial resampling where there is overlap in train and test data.
+% #| fig-alt: Six scatter plots are shown with y-axes from 4 degrees South to 3.97 degrees South and x-axes from 79.08 degrees West to 79.06 degrees West. The top row of plots show a very clear separation with no overlap between blue and orange datapoints, whereas the bottom low shows significant overlap.
+% #| label: fig-sprsmp
+%
+% library(patchwork)
+% autoplot(rsmp_sp, task, fold_id = c(1:3), size = 0.7) /
+%   autoplot(rsmp_nsp, task, fold_id = c(1:3), size = 0.7) &
+%   scale_y_continuous(breaks = seq(-3.97, -4, -0.01)) &
+%   scale_x_continuous(breaks = seq(-79.06, -79.08, -0.02))
+% ```
+%
+% Now we have seen why spatial resampling methods matter we can take a look at what is implemented in `r mlr3spatiotempcv`.
+% At the time of writing we have implemented 19 different resampling methods for spatial and spatiotemporal analysis.
+% These are categorized into:
+%
+% * Blocking -- Create rectangular blocks in 2D/3D space (as opposed to partitioning in 1D space used in non-spatial methods)
+% * Buffering -- Create buffering zones to remove observations between train and test sets
+% * Spatiotemporal clustering -- Clusters based on coordinates (and/or time-points)
+% * Feature space clustering -- Clusters based on feature space and not necessarily spatiotemporal
+% * Partitioning -- Grouped by factor variables
+%
+% The choice of method may depend on specific characteristics of the dataset and there is no easy rule to pick one method over another, full details of different methods can be found in @Schratz2021 -- the paper deliberately avoids recommending one method over another as the 'optimal' choice is highly dependent on the predictive task, autocorrelation in the data, and the spatial structure of the sampling design.
+% The documentation for each of the implemented methods includes details of each method as well as references to original publications.
+%
+% :::{.callout-tip}
+% ## Spatio*temporal* resampling
+%
+% We have focused on spatial analysis but referred to "spatiotemporal" and "spatiotemp".
+% The resampling methods discussed in this section extend trivially to temporal analysis (or spatial and temporal analysis combined) by setting the `"time"` `col_role` in the task (@sec-performance).
```

Comment:
```suggestion
% The spatial-only resampling methods discussed in this section extend trivially to temporal analysis (or spatial and temporal analysis combined) by setting the `"time"` `col_role` in the task (@sec-performance).
```

Reply:
Reading this as a user, I would be a bit confused:

- What does it mean here "extend trivially"? What would I need to do exactly? (Disclaimer: I don't think it's "that easy")
- Defining `time` as a col_role is only helpful (and needed) when aiming for a spatiotemporal partitioning, which is quite advanced as the partitioning gets a lot more complex.

I'd suggest to reword and take out "trivally" and mention that so far everything shown has focused on spatial-only partitioning and extending it to spatiotemporal is possible but advanced.